pandas_gbq/gbq.py

```diff
@@ -1050,12 +1088,7 @@ def to_gbq(
         )
 
     if api_method == "default":
-        # Avoid using parquet if pandas doesn't support lossless conversions to
```

Comment:
per coverage report, this was dead code. Our minimum pandas version is beyond the one where this feature wasn't available.

Reply:
thank you this is a helpful comment!